src/subscription/checkout/SubscriptionCheckout.jsx

```diff
@@ -32,18 +31,17 @@ export const SubscriptionCheckout = () => {
     isSubscriptionDetailsProcessing,
     paymentMethod,
   } = useSelector(subscriptionSelector);
-  const { clientSecretId, isClientSecretProcessing } = useSelector(getClientSecretSelector);
+  // const { clientSecretId, isClientSecretProcessing } = useSelector(getClientSecretSelector);
```

Comment:
You still want to keep this commented out?

Reply:
removed

Reply:
in next follow up PR.